svclib/types.go

```diff
@@ -1,5 +1,7 @@
 package svclib
 
+import "rules_itest/logger"
```

Comment:
I'd slightly prefer not introducing this dependency, I was potentially planning to have an importable Go client that can parse ports from the env. But we can rearrange later if we decide to do that, not a huge deal.